docs/data/events.md

```diff
@@ -68,16 +68,16 @@ An Event may have an Evidence Record associated with it. This is a file that pro
 The Event may include metadata about its subject and/or object. For example:
 
  - `pid` - The persistent ID. Must correspond to `subj_id` or `obj_id`.
- - `issued` - publication date.
- - `title` - the title of the webpage, comment, etc.
- - `author` - author of the comment, blog etc.
+ - `issued` - Publication date.
+ - `title` - The title of the webpage, comment, etc.
+ - `author` - Author of the comment, blog etc.
  - `url` - URL where this was found. May be different to `pid`.
- - `method` - Optional. The method by which we matched a URL for Registered Content to a DOI.
+ - `method` - Optional. The method by which we matched a URL for registered content to a DOI.
  - `verification`. Optional. The method by which we verified that this content has the given DOI.
 
 We typically only include metadata that you are unable to get elsewhere and that is unlikely to change. Where the `subj_id` or `obj_id` is a DOI, we will not include the metadata because you can easily look it up using the Crossref or DataCite APIs and it could go out of date if recorded. We also store a large amount of metadata associated with each DOI, and including it all would balloon the data to many times its size.
 
-People don't always use DOIs to cite Registered Content, especially on social media, so we often links using the Landing Page. Where a link to Landing Page is identified, and we can match that to a DOI, we will use the DOI as the `obj_id` and `obj.pid`, and the actual URL of the Landing Page as the `obj.url`. In these cases, we include an optional `method` and `verification` and field to indicate how we matched a DOI, and how we verified that. For a full description see [Matching Landing Pages](/data/matching-landing-pages).
+People don't always use DOIs to cite registered content, especially on social media, so we often links using the landing page. Where a link to landing page is identified, and we can match that to a DOI, we will use the DOI as the `obj_id` and `obj.pid`, and the actual URL of the landing page as the `obj.url`. In these cases, we include an optional `method` and `verification` and field to indicate how we matched a DOI, and how we verified that. For a full description see [Matching Landing Pages](/data/matching-landing-pages).
```

Comment:
'Matching Landing Pages' is a page title, so presumably should follow the other edits, which put titles in sentence case.